pkg/agent/controller/agent.go

```diff
@@ -146,6 +158,18 @@ func NewWithDetail(spec *AgentSpecification, syncerConf *broker.SyncerConfig, re
 		return nil, err
 	}
 
+	agentController.serviceLHExportController, err = newLHServiceExportController(lighthouseClient, mcsClient)
+
+	if err != nil {
+		return nil, err
+	}
+
+	agentController.serviceMCSExportController, err = newMCSServiceExportController(lighthouseClient, mcsClient)
+
```

Comment:
We should use ResourceSyncers - saves a ton of boilerplate.

Reply:
Replaced with resourceSyncer, there is a UT failure "no kind is registered for the type v2alpha1.ServiceExport", I think I need to add it to scheme somewhere in the UT.

Reply:
yes - L354 in _agent_test.go_.

Reply:
Thanks I have fixed it .